README.md

```diff
@@ -116,7 +126,9 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+$("#greeting").on("click", function (e) {
+  $(this).append("<p>hello</p>");
+});
 ```
```

Comment:
This code will append hello to the button with id greeting. We just need to replace `this` with `"body"`